packages/docusaurus/src/webpack/utils.ts

```diff
@@ -326,8 +326,9 @@ type FileLoaderUtils = {
 
 // Inspired by https://github.com/gatsbyjs/gatsby/blob/8e6e021014da310b9cc7d02e58c9b3efe938c665/packages/gatsby/src/utils/webpack-utils.ts#L447
 export function getFileLoaderUtils(): FileLoaderUtils {
-  // files/images < 10kb will be inlined as base64 strings directly in the html
-  const urlLoaderLimit = 10000;
+  // files/images < 10kb (overridable via 'URL_LOADER_LIMIT' environment
+  // variable) will be inlined as base64 strings directly in the html
+  const urlLoaderLimit = process.env.URL_LOADER_LIMIT || 10000;
```

Comment:
Don't know if it would be a great deal but could we have `URL_LOADER_LIMIT=0` to make all images not inlined (as in your use-case)? Won't that break this logic?

```suggestion
  const urlLoaderLimit = process.env.URL_LOADER_LIMIT ?? 10000;
```

Reply:
I don't think zero makes sense in a prod env, but fair point. LGTM.

Reply:
One can also discuss if 10000 is a resonable default :) I'd set it at 1000 or max 2000 personally.

Reply:
Just wondering because your intention of this PR _is_ to make all images not inlined :P Won't the setting be `URL_LOADER_LIMIT=0`?

Reply:
As a reference mod_pagespeed uses 2k:
https://www.modpagespeed.com/doc/reference-image-optimize#ImageInlineMaxBytes

Reply:
Oh okay, I see the point now

Reply:
I think your improvement is good. I started the default size discussion to the issue.